index.html

```diff
@@ -3177,6 +3177,273 @@ <h3>Definition of Roles</h3>
 						</tbody>
 					</table>
 				</div>
+        <div class="role">
+          <rdef>doc-pagefooter</rdef>
+          <div class="role-description">
+            <p>A repeated section of text that is separated from the body text and appears at the bottom of a page, containing items such as the page number, footnotes, and repeated items such a shortened document title or the author's last name.</p>
+            <p>The doc-pagefooter role MUST be used on every instance of the page footer.</p>
+            <p>The doc-pagefooter MUST NOT be used to contain the doc-pagebreak.</p>
+            <p>Assistive technologies MAY allow the user to read the document content continuously without interruption from the page footer.</p>
+            <pre class="example highlight">&lt;section role="doc-pagefooter"&gt;
+   &lt;div&gt;War and Peace&lt;/h2&gt;
+   &lt;div role="doc-pagenum"&gt;321&lt;/div&gt;
+   &#8230;
+&lt;/section&gt;</pre>
+          </div>
+          <table class="role-features">
+            <caption>Characteristics:</caption>
+            <thead>
+              <tr>
+                <th scope="col">Characteristic</th>
+                <th scope="col">Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th class="role-abstract-head" scope="row">Is Abstract:</th>
+                <td class="role-abstract"> </td>
+              </tr>
+              <tr>
+                <th class="role-parent-head" scope="row">Superclass Role:</th>
+                <td class="role-parent"><rref>section</rref></td>
+              </tr>
+              <tr>
+                <th class="role-children-head" scope="row">Subclass Roles:</th>
+                <td class="role-children"> </td>
+              </tr>
+              <tr>
+                <th class="role-base-head" scope="row">Base Concept:</th>
+                <td class="role-base"> </td>
+              </tr>
+              <tr>
+                <th class="role-related-head" scope="row">Related Concepts:</th>
+                <td class="role-related">EPUB <a
```

Comment:
No equivalent for running footers in the SSV.

Reply:
Fixed